sdk/avalon_sdk/fabric/eventlistener.py

```diff
@@ -84,16 +87,18 @@ def saveConfig(self):
 
     async def startEventHandling(self):
```

Comment:
Please use PEP8 for function names. startEventHandling -> start_event_handling

Reply:
Why can't we enable pep8 standard in lint to catch these errors?

Reply:
We can by following http://pycodestyle.pycqa.org/en/latest/intro.html. But we need to ensure that existing files are following PEP8 top avoid breaking the build due to lint issues.  So first we need to raise a separate PR fixing PEP8 issues in existing files.

Reply:
@Ram-srini: we can enable PEP8 in `run_lint`. The problem is the current code is not PEP8 compliant and requires fixing.

Reply:
Done.